### ISSUE TITLE
What are tables and how do we use them?

### ISSUE THREAD

**Issue by waldoj** [openva/crump#18]
Throughout the [file layouts](https://www.scc.virginia.gov/clk/files/layout_be.pdf), there are references to "Table Codes." For example, in [the ninth file](https://github.com/openva/crump/blob/master/table_maps/9_llc.yaml), there are several fields that are "table codes," like "LLC Status" and "State of Formation."

I'm pretty sure that the lookup tables are all listed in the first file. Here's an excerpt:

``` csv
01,00,00,TABLE NAMES
01,00,01,CORPORATE STATUS TABLE
01,00,02,STATE TABLE
01,00,03,INDUSTRY CODE TABLE
01,00,04,REGISTERED AGENT STATUS
01,00,05,COURT LOCALITY CODE
01,00,06,ANNUAL REPORT STATUS
01,00,07,ASSESSMENT INDICATOR
01,00,12,STOCK CLASS
01,00,19,AMENDMENT TYPES
01,00,23,L/P AMENDMENT & L/P CERT. OF CORRECTION TYPES
01,00,28,L/P REGISTERED AGENT STATUS
01,00,40,LLC REGISTERED AGENT STATUS
01,01,00,ACTIVE
01,01,01,FEE DELINQUENT
01,01,14,DISSOLVING
01,01,15,CANC(VOLUNTARY)  (LLC/LP/BT)
01,01,16,INVDISS/COURT
01,01,17,CANC/DOM         (CORP)
01,01,18,SURRENDERED      (CORP)
01,01,19,CANC (AUTO-FEE) LLC/LP/BT-FEES NOT PAID
01,01,20,MERGED
01,02,AK,ALASKA
01,02,AL,ALABAMA
01,02,AR,ARKANSAS
01,02,AS,AMERICAN SAMOA
01,02,AZ,ARIZONA
01,02,CA,CALIFORNIA
```

The first column—`01`—is just the file ID. We can ignore that. I'm pretty sure that the second one is the lookup table ID, which appears to resolve to [the list of table IDs](https://github.com/openva/crump/blob/master/table_types.csv).

Basically, I think this is a metadata table with listing foreign keys.
